app/models/wiki.rb

```diff
@@ -96,7 +101,7 @@ def self.default_wiki
   end
 
   def self.get_or_create(language:, project:)
-    language = nil if MULTILINGUAL_PROJECTS.include?(project)
+    language = nil if project == 'wikidata'
```

Comment:
Per above comment, I guess this is the place where we'd want to convert 'www' to `nil`. (Doing it in `#ensure_valid_project` as well doesn't hurt, especially since we use FactoryGirl rather than this `.get_or_create` method in some of the specs.)